desktop/apps/consignments2/components/choose_artist/index.js

```diff
@@ -54,32 +54,37 @@ function ChooseArtist (props) {
 
   return (
     <div className={b()}>
-      <div className={b('label')}>Artist/Designer Name</div>
-      <div className={b('autosuggest')}>
-        <Autosuggest
-          suggestions={artistAutocompleteSuggestions}
-          onSuggestionsFetchRequested={fetchArtistSuggestionsAction}
-          onSuggestionsClearRequested={clearArtistSuggestionsAction}
-          onSuggestionSelected={chooseArtistAndAdvanceAction}
-          getSuggestionValue={getSuggestionValue}
-          renderInputComponent={renderInputComponent}
-          renderSuggestion={renderSuggestion}
-          inputProps={inputProps}
-        />
+      <div className={b('title')}>
```

Comment:
Looks like a big diff but I just added the `title` to this component and nested everything below.